### PR TITLE
job.md: add section on suspended jobs

### PR DESCRIPTION
**Pull request by adtac**
Replaces https://github.com/kubernetes/website/pull/26398

>KEP issue: https://github.com/kubernetes/enhancements/issues/2232
>KEP: https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/2232-suspend-jobs
>Implementation: https://github.com/kubernetes/kubernetes/pull/98727

/sig apps
/hold until https://github.com/kubernetes/kubernetes/pull/98727 is merged
/cc @ahg-g @alculquicondor @soltysh @reylejano

### CODE REVIEW

**Review by irvifa — COMMENTED** [review]
/milestone 1.21

**Review by soltysh — APPROVED** [review]
/lgtm
from tech pov

**Comment by adtac** [comment]
- added note about the semantics of ADS/startTime wrt suspend
- added a note about the feature gate

File: content/en/docs/concepts/workloads/controllers/job.md
Comment (alculquicondor) [inline content/en/docs/concepts/workloads/controllers/job.md:473]: Maybe exemplify with a parallelism of 2? Otherwise too many lines.

But I think `Events are created when the Job is suspended and resumed` should be enough. Users should know how to look at events.

**Comment by alculquicondor** [comment]
@reylejano note that https://github.com/kubernetes/website/pull/26466#pullrequestreview-598962163 constitutes the review from sig-apps

**Comment by adtac** [comment]
Squashed commits.

**Review by soltysh — APPROVED** [review]
/lgtm

File: content/en/docs/concepts/workloads/controllers/job.md
Comment (annajung) [inline content/en/docs/concepts/workloads/controllers/job.md:384]: If behind the feature gate, you need to add it to Alpha/Beta Feature gates table https://github.com/kubernetes/website/blob/master/content/en/docs/reference/command-line-tools-reference/feature-gates.md#feature-gates-for-alpha-or-beta-features

Comment (adtac) [reply content/en/docs/concepts/workloads/controllers/job.md:384]: thanks for catching that, I've added row

Comment (annajung) [reply content/en/docs/concepts/workloads/controllers/job.md:384]: could you make sure that you also have a description for the feature https://github.com/kubernetes/website/blob/master/content/en/docs/reference/command-line-tools-reference/feature-gates.md#list-of-feature-gates-feature-gates?

Comment (adtac) [reply content/en/docs/concepts/workloads/controllers/job.md:384]: Just caught that a second after I pushed :) fixed.

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-26466--kubernetes-io-vnext-staging.netlify.app/docs/concepts/workloads/controllers/job/#suspending-a-job

**Comment by reylejano** [comment]
Hi @adtac, This PR has a tech review & lgtm and a sig docs member review but can you squash your commits?

This PR is on track for 1.21, just wanted to share upcoming doc related dates for the 1.21 release:
- <strike> Docs Ready for Review deadline is March 24</strike> done
- Docs Ready to Merge deadline is March 31

Thank you

**Comment by adtac** [comment]
@reylejano squashed and rebased. PTAL, thanks!

**Comment by sftim** [comment]
Also see PR #27197

File: content/en/docs/concepts/workloads/controllers/job.md
Comment (sftim) [inline content/en/docs/concepts/workloads/controllers/job.md:466]: This is YAML, not JSON.

File: content/en/docs/concepts/workloads/controllers/job.md
Comment (sftim) [inline content/en/docs/concepts/workloads/controllers/job.md:408]: I would use a different anchor _and_ heading here
```suggestion
### Job suspension {#field-spec-suspend}
```

**Review by sftim — COMMENTED** [review]
Also LGTM, but I did spot one quibble.

File: content/en/docs/concepts/workloads/controllers/job.md
Comment (sftim) [reply content/en/docs/concepts/workloads/controllers/job.md:408]: (not vital, doesn't block merge)

**Comment by sftim** [comment]
/approve

I'll send in a follow up PR

**Pull request merged**
